data/json/npcs/refugee_center/surface_refugees/NPC_Fatima_Al_Jadir.json

```diff
@@ -0,0 +1,173 @@
+[
+  {
+    "type": "npc",
+    "id": "refugee_FatimaalJadir",
+    "name_unique": "Fatima al Jadir",
```

Comment:
Shouldn't it be `Fatima al-Jadir`?

Reply:
Nope, that's a different name with a different meaning